js/app.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+(function(){
+  angular
+  .module("radioLab", [
+    "ui.router",
+    "ng-token-auth",
+    "songs"
+  ])
+  .config([
+    "$authProvider",
+    AuthConfigFunction
+  ])
+  .config([
```

Comment:
If you have multiple configs you could put them in separate files. It'd be a bit more Angular-y.